yourgov/components/FormMobileFoodVendorPermit/task1/FormTask1Step1.tsx

```diff
@@ -0,0 +1,161 @@
+	SummaryListItem,
+	SummaryListItemDescription,
+	SummaryListItemTerm,
+} from '@ag.ds-next/react/summary-list';
+import { TextInput } from '@ag.ds-next/react/text-input';
+import { StepActions } from '../StepActions';
+import { useGlobalForm } from '../GlobalFormProvider';
+import {
+	task1Step1Part2FormSchema,
+	type Task1Step1Part2FormSchema,
+} from './FormTask1FormState';
+import { FormTask1Container } from './FormTask1Container';
+import { task1FormSteps, useFormTask1Context } from './FormTask1Provider';
+
+export function FormTask1Step1() {
+	const { formState } = useGlobalForm();
+	const stepFormState = formState.task1?.step1;
+	const { query } = useRouter();
+	const isUpdated = query.success === 'true';
+	const [isSuccessMessageVisible, setIsSuccessMessageVisible] =
+		useState(isUpdated);
+
+	useEffect(() => {
+		setIsSuccessMessageVisible(isUpdated);
+	}, [isUpdated]);
+
+	return (
+		<FormTask1Container
+			formTitle="Owner details"
+			formIntroduction="Confirm your name and contact details."
+			shouldFocusTitle={!isSuccessMessageVisible}
+		>
+			<Stack gap={3}>
+				<Stack gap={1.5}>
+					<H2>Confirm business owner details</H2>
+					<Details label="How were my details prefilled?" iconBefore>
+						<Prose>
+							<p>
+								We’re working hard to improve the way we do business with you.
+								This includes making applications and registrations easier to
+								use.
+							</p>
+							<p>
+								If we already have some of the information you need to tell us,
+								we’ll pre-fill it into your applications. This saves you
+								entering all your details yourself.
+							</p>
+							<p>
+								It’s important to check the pre-filled information in your
+								report before you submit it.
+							</p>
+						</Prose>
+					</Details>
+					{isSuccessMessageVisible && (
+						<SectionAlert
+							focusOnMount
+							title="Business owner details have been updated"
+							tone="success"
+							onClose={() => setIsSuccessMessageVisible(false)}
+						/>
+					)}
+					<Stack gap={1.5} alignItems="flex-start">
+						<SummaryList>
+							<SummaryListItem>
+								<SummaryListItemTerm>First name</SummaryListItemTerm>
+								<SummaryListItemDescription>
+									{stepFormState?.firstName}
+								</SummaryListItemDescription>
+							</SummaryListItem>
+							<SummaryListItem>
+								<SummaryListItemTerm>Last name</SummaryListItemTerm>
+								<SummaryListItemDescription>
+									{stepFormState?.lastName}
+								</SummaryListItemDescription>
+							</SummaryListItem>
+							<SummaryListItem>
+								<SummaryListItemTerm>Email address</SummaryListItemTerm>
+								<SummaryListItemDescription>
+									{stepFormState?.email}
+								</SummaryListItemDescription>
+							</SummaryListItem>
+						</SummaryList>
+						<ButtonLink
+							href={task1FormSteps[0].items && task1FormSteps[0].items[0].href}
+							variant="text"
+						>
+							Change business owner details
+						</ButtonLink>
```

Comment:
Should the condition wrap the whole link?

```suggestion
						{task1FormSteps[0].items && (
							<ButtonLink href={task1FormSteps[0].items[0].href} variant="text">
								Change business owner details
							</ButtonLink>
						)}
```

Reply:
It's only there to appease TS right? The first item is never not going to have an href in reality